### ISSUE TITLE
[AdminBundle] Range slider not styled correctly + add value indication with js

### ISSUE THREAD

**Issue by jockri** [Kunstmaan/KunstmaanBundlesCMS#473]
![screenshot 2015-06-04 15 39 57](https://cloud.githubusercontent.com/assets/517574/7985302/13dfec22-0ad0-11e5-8cb2-b26b129b28d9.png)

Example code for form type:
```php
    $builder->add('percentage', 'range', array(		
        'attr' => array('min' => 0, 'max' => 100)		
    ));
```

**Comment by jockri** [comment]
@FVKVN Did you already do this?

**Comment by diskwriter** [comment]
fixed in #954
